packages/expo-device/src/Device.ts

```diff
@@ -98,6 +98,7 @@ export const deviceType: DeviceType | null = ExpoDevice ? ExpoDevice.deviceType
 
 /**
  * The [device year class](https://github.com/facebook/device-year-class) of this device. On web, this value is always `null`.
+ * @deprecated Since the official Facebook Device Year Class library is deprecated, this constant is also deprecated and will be removed in a future SDK version.
```

Comment:
Can we link to the archived repo?
```suggestion
 * @deprecated Since the official [Facebook Device Year Class](https://github.com/facebookarchive/device-year-class) library is deprecated, this constant is also deprecated and will be removed in a future SDK version.
```